tests/config/testconfig.yaml

```diff
@@ -1,5 +1,6 @@
 clusterName: "staging-usw2-k8s"
 syncTimeIntervalMs: 30000
+logLevel: 1
 cascadeDeletion: false
```

Comment:
don't we need to purge the fields from this that we're no longer using?

Reply:
also please update file below, plus [this](https://github.com/numaproj/numaplane/blob/main/config/manager/controller-config.yaml#L7) file

Reply:
+1